tests/e2e/android-kotlin/readme.md

```diff
@@ -0,0 +1,46 @@
+# RudderTyper - Android Kotlin
+
+This is a sample Android Kotlin app designed to demonstrate how to use the RudderStack Android SDK to track the RudderTyper generated events.
+
+## Prerequisites
+
+- Download, install and setup [Android Studio](https://developer.android.com/studio/install).
+
+## Getting Started
+
+### Step 1: Open the `android-kotlin` folder, located under `tests/e2e`, in Android Studio
+
+### Step 2: Configure RudderStack Properties
+- Rename `rudderstack.properties.sample` to `rudderstack.properties`.
+- Update the `writeKey`, `dataPlaneUrl`, and `controlPlaneUrl` with your specific values.
+
+### Step 3: Sync Gradle
+
+- Sync the Gradle files by clicking on the `Sync Now` link in the top right corner of the Android Studio window. Or, you can click on the `File` menu and select `Sync Project with Gradle Files`:
+
+    ![syncGradle](images/syncGradle.png)
+
+### Step 4: Add Generated Files
+- Crete a new package `ruddertyper` under `app/src/main/java/com/rudderstack/typewriterexample`:
+    ![createNewRudderTyperPackage.png](images/createNewRudderTyperPackage.png)
+- Place the RudderTyper-generated files in the `ruddertyper` package.
+
+### Step 5: Implement Event Tracking
```

Comment:
Some basic sample event tracking should be automatically part of the app. Can we add it if not already?

Reply:
Adding sample events requires the RudderTyper-generated files to be present in the sample app, which is not currently included. Therefore, I’ve provided instructions in the README as well as [here](https://github.com/rudderlabs/rudder-typer/blob/9f8835cb5200641ec614b5d7f0af2d8c05e7e314/tests/e2e/android-kotlin/app/src/main/java/com/rudderstack/typewriterexample/MainActivity.kt#L55).

Reply:
The sample events can get autogenerated from 
`tests/e2e/android-java/app/src/main/java/com/segment/generated/plan.json` <- deleted file
we can have a sample schema here with sample naming. <- the `npx ruddertyper prod/dev` first looks for schema in plan.json if this file is not there then only it fetches from API.

Reply:
@1abhishekpandey, is this also done now with your recent changes?

Reply:
(I was going to comment.)
I've added the sample events in the app, please find it [here](https://github.com/rudderlabs/rudder-typer/blob/799b84e7439362c1b26570d86e2a71eca5907aef/tests/e2e/android-kotlin/app/src/main/java/com/rudderstack/ruddertyper/MainActivity.kt#L57).